client/packages/executor/src/executionManager/sideEffect.ts

```diff
@@ -397,6 +407,26 @@ export class SideEffect extends EventEmitter {
         this.triggerBid()
     }
 
+    /**
+     * Keep a record of how much a certain executor bid on the SFX.
+     * 
+     * @param xtxId ID of the executor
+     * @param bid The amount the executor bid
+     */
+    storeXtxBidHistory(xtxId: string, bid: number) {
```

Comment:
xtxId is the name for the id of the XTX object, not the executor (each Execution class instance has an xtxId which is generated by the circuit). Please rename accordingly!

Reply:
Done! ✌️
I messed up names, I don't know why. Removed the method there because it's already present in the bidding engine.